shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/BaseRule.g4

```diff
@@ -1808,10 +1808,8 @@ relationExprList
     ;
 
 relationExpr
-    : qualifiedName
-    | qualifiedName ASTERISK_
-    | ONLY qualifiedName
-    | ONLY LP_ qualifiedName RP_
+    : tableName (ASTERISK_)?
```

Comment:
Why change qualifiedName to tableName?

Reply:
I chose tableName because it matches with the syntax name provided in the docs

Reply:
https://www.postgresql.org/docs/current/sql-alterpublication.html
Do you think we should keep it as qualifiedName?

Reply:
Hi @everly-gif Can you take a look at PostgreSQL SQL parser source code? We need to be consistent with it to ensure grammatical correctness.

Following is the syntax definition in the PostgreSQL gram.y file:

```
relation_expr:
			qualified_name OptSnapshotVersion
				{
					/* default inheritance */
					$$ = $1;
					if ($2 != NULL)
					{
						char *snapshot_name = (char *)palloc0(strlen($1->relname) + 1 + strlen($2) + 1);
						sprintf(snapshot_name, "%s%c%s", $1->relname, DB4AI_SNAPSHOT_VERSION_DELIMITER, $2);
						$$->relname = snapshot_name;
					}
					$$->inhOpt = INH_DEFAULT;
					$$->alias = NULL;
				}
			| qualified_name '*'
				{
					/* inheritance query */
					$$ = $1;
					$$->inhOpt = INH_YES;
					$$->alias = NULL;
				}
			| ONLY qualified_name
				{
					/* no inheritance */
					$$ = $2;
					$$->inhOpt = INH_NO;
					$$->alias = NULL;
				}
			| ONLY '(' qualified_name ')'
				{
					/* no inheritance, SQL99-style syntax */
					$$ = $3;
					$$->inhOpt = INH_NO;
					$$->alias = NULL;
				}
		;


relation_expr_list:
			relation_expr							{ $$ = list_make1($1); }
			| relation_expr_list ',' relation_expr	{ $$ = lappend($1, $3); }
		;
```

Reply:
Thank you for sharing, changing it back to qualifiedName